cmd/gpu_plugin/README.md

```diff
@@ -50,6 +55,74 @@ The following sections detail how to obtain, build, deploy and test the GPU devi
 
 Examples are provided showing how to deploy the plugin either using a DaemonSet or by hand on a per-node basis.
 
+### Prerequisites
+
+Access to a GPU device requires firmware, kernel and user-space
+drivers supporting it.  Firmware and kernel driver need to be on the
+host, user-space drivers in the GPU workload containers.
+
+GPU devices supported by the current kernel can be listed with:
+```
+head /sys/class/drm/card[0-9]/device/uevent
+```
+
+Ones with `DRIVER=i915` are Intel ones supported by this plugin. GPUs
+with display support belong to `PCI_CLASS=30000`, and ones without
+display to `PCI_CLASS=38000`:
+```
+==> /sys/class/drm/card0/device/uevent <==
+DRIVER=i915
+PCI_CLASS=38000
+PCI_ID=8086:56C0
+PCI_SUBSYS_ID=8086:4905
+PCI_SLOT_NAME=0000:4d:00.0
+MODALIAS=pci:v00008086d000056C0sv00008086sd00004905bc03sc80i00
+```
+
+For the integrated (older) GPUs, new enough firmware and kernel driver
+are typically included already with the host OS, and new enough
+user-space drivers are in the host OS repositories (used for building
+the containers).
```

Comment:
I'd move these under the sub topic below since it's anyway talking about drivers, firmware and user space drivers.

Reply:
Topic below is about dGPUs, this text about iGPUs.  Did you mean adding "Integrated GPU drivers" section heading for this paragraph?

Reply:
Hmm.. maybe a new section for "Integrated GPU drivers" below the "Discrete GPU drivers" (on the same level). It doesn't have to so thorough because, as you note, drivers and firmware are already part of most distributions.

Reply:
Added separate "Drivers for older (integrated) GPUs" section after discrete one.